homeassistant/components/homeassistant_connect_zbt2/config_flow.py

```diff
@@ -103,6 +105,21 @@ async def async_step_install_thread_firmware(
             next_step_id="finish_thread_installation",
         )
 
+    def _extra_zha_hardware_options(self) -> dict[str, Any]:
+        """Return extra ZHA hardware options."""
+        country = self.hass.config.country
+
+        if country is None:
+            tx_power = RADIO_TX_POWER_DBM_DEFAULT
+        else:
+            tx_power = RADIO_TX_POWER_DBM_BY_COUNTRY.get(
+                country, RADIO_TX_POWER_DBM_DEFAULT
+            )
```

Comment:
Is the `if country is None` just done for clarity (or speed)?

Otherwise, wouldn't this be enough, since the default for non-existent keys (like `None`) is `RADIO_TX_POWER_DBM_DEFAULT` already?
```suggestion
        tx_power = RADIO_TX_POWER_DBM_BY_COUNTRY.get(
            country, RADIO_TX_POWER_DBM_DEFAULT
        )
```